frontend/src/Screens/GlobalComponents/Navbar/Components/PageDesign/ProjectCards/ProjectCards.css

```diff
@@ -0,0 +1,21 @@
+.Cards__container{
+    
+}
+.Cards__Card{
+    background-color: var(--white);
+    color: black;
+    width: 25rem;
```

Comment:
please change to px

Reply:
The file ProjectCard.css does not exist.